digit-recognition/frontend/src/features/digit-recognizer/utils.ts

```diff
@@ -19,18 +19,26 @@ export const getFlattenedPixelArray = (
 
   const imageData = tempCtx.getImageData(0, 0, expectedSize, expectedSize);
 
-  const pixels = imageData.data; // Array of RGBA values ​​(4 elements per pixel)
+  const pixels = imageData.data; // Array of RGBA values (4 elements per pixel)
   const grayscaleArray = [];
 
   for (let i = 0; i < pixels.length; i += 4) {
-    const r = pixels[i];
-    const g = pixels[i + 1];
-    const b = pixels[i + 2];
-    const brightness = Math.round((r + g + b) / 3);
-
-    grayscaleArray.push(brightness > 0 ? 255 : 0);
+    if (pixels[i] === 0) {
+      grayscaleArray.push(0);
+      continue;
+    }
+
+    const getRandomDeviation = () => Math.round(Math.random() * 10 - 5);
```

Comment:
Why ‘* 10 - 5’ is needed? Better to name them as a constants if they are not random values.